src/axolotl/utils/models.py

```diff
@@ -450,11 +452,11 @@ def load_llama_adapter(model, cfg):
         task_type="CAUSAL_LM",
     )
 
-    if cfg.lora_model_dir:
+    if cfg.peft_model_dir or cfg.lora_model_dir:
```

Comment:
Since we're updating to `peft_model_dir`, we could add a deprecation warning to validate config to reduce need for checking both like this line.

For backward compatibility, we can assign `cfg.peft_model_dir = cfg.lora_model_dir` if it's not None.